adapter/network_management.md

```diff
@@ -251,3 +274,55 @@ Every adapter should define its own service name.
   "topic": "pt:j1/mt:evt/rt:ad/rn:zw/ad:1"
 }
 ```
+
+* Example of a command requesting ping for thing at address 4:
+
+```json
+{
+  "serv": "zwave-ad",
+  "type": "cmd.ping.send",
+  "val_t": "string",
+  "val": "4",
+  "props": null,
+  "tags": null,
+  "src": "-",
+  "ver": "1",
+  "ctime": "2018-11-22T23:14:40+0100",
+  "uid": "1e965f4c-07ee-4e3e-8c03-e61e9aa9192a",
+  "topic": "pt:j1/mt:cmd/rt:ad/rn:zw/ad:1"
+}
+```
+
+* Example of a report containing ping report for thing 4 connected indirectly through single hop at address 2:
+
+```json
+{
+  "serv": "zwave-ad",
+  "type": "evt.ping.report",
+  "val_t": "object",
+  "val": {
```

Comment:
What about being backward compatible and having more proeprties here for FimpUI usage as an example?
We want to keep it backward compatible, but we do not want to state it in FIMP specs?

Reply:
Yes. Leave just comments. Also Angry dog uses only success/failure and only TP flow is using all the bs left in Z-Wave message.